### ISSUE TITLE
Adding articles is slow

### ISSUE THREAD

**Issue by Strubbl** [TheFive/osmbc#1044]
1. go to https://osmbc.openstreetmap.de/article/create
2. add a new article URL and hit Search button
3. select a category for the new article and hit OK button
4. Now the site loads approx. one minute or longer to add the article to the WN

**Comment by TheFive** [comment]
still an issue ?

**Comment by Strubbl** [comment]
Yes, still an issue. You can see this, when you add an article. In the moment you hit the OK button in the search & create view (e.g. https://osmbc.openstreetmap.de/article/searchandcreate?search=https%3A%2F%2Fosmbc.openstreetmap.de%2Farticle%2Fcreate&SearchNow=), you can already see the message in Mattermost in the channel osmbcarticle, but the osmbc site itself is still loading like 5 secs more.

**Comment by Strubbl** [comment]
JFYI: Looking back at this issue, i think it is really solved. I haven't experience any bad performance when adding an article anymore in the last weeks.
